src/main/java/teammates/ui/controller/InstructorFeedbackRemindParticularStudentsAction.java

```diff
@@ -31,12 +31,12 @@ protected ActionResult execute() {
         taskQueuer.scheduleFeedbackSessionRemindersForParticularUsers(courseId, feedbackSessionName, usersToRemind);
 
         statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT, StatusMessageColor.SUCCESS));
-        statusToAdmin = "Email sent out to the selected user(s): ";
+        statusToAdmin.add("Email sent out to the selected user(s): ");
         for (String user : usersToRemind) {
-            statusToAdmin += "<br>" + user;
+            statusToAdmin.add("<br>" + user);
```

Comment:
Think the original approach of building up a status message then appending it to statusToAdmin is better, since each message is one status, but these are part of the same status message.